specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2024-05-15/managedCassandra.json

```diff
@@ -0,0 +1,1426 @@
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Certificate"
+              }
+            },
+            "externalGossipCertificates": {
+              "description": "List of TLS certificates used to authorize gossip from unmanaged data centers. The TLS certificates of all nodes in unmanaged data centers must be verifiable using one of the certificates provided in this property.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Certificate"
+              }
+            },
+            "gossipCertificates": {
+              "readOnly": true,
+              "x-ms-mutability": [
+                "read"
+              ],
+              "description": "List of TLS certificates that unmanaged nodes must trust for gossip with managed nodes. All managed nodes will present TLS client certificates that are verifiable using one of the certificates provided in this property.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/Certificate"
+              }
+            },
+            "externalSeedNodes": {
+              "type": "array",
+              "description": "List of IP addresses of seed nodes in unmanaged data centers. These will be added to the seed node lists of all managed nodes.",
+              "items": {
+                "$ref": "#/definitions/SeedNode"
+              }
+            },
+            "seedNodes": {
+              "readOnly": true,
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/SeedNode"
+              },
+              "description": "List of IP addresses of seed nodes in the managed data centers. These should be added to the seed node lists of all unmanaged nodes."
+            },
+            "hoursBetweenBackups": {
+              "type": "integer",
+              "format": "int32",
+              "description": "(Deprecated) Number of hours to wait between taking a backup of the cluster."
+            },
+            "deallocated": {
+              "type": "boolean",
+              "description": "Whether the cluster and associated data centers has been deallocated."
+            },
+            "cassandraAuditLoggingEnabled": {
+              "type": "boolean",
+              "description": "Whether Cassandra audit logging is enabled"
+            },
+            "provisionError": {
+              "$ref": "#/definitions/CassandraError",
+              "description": "Error related to resource provisioning."
+            },
+            "azureConnectionMethod": {
+              "type": "string",
+              "description": "How to connect to the azure services needed for running the cluster",
```

Comment:
>azure [](http://example.com/codeflow?start=52&length=5)

isn't style guide to capitalize Azure?

Reply:
Will take it in a later change, related to managedCassandra nits.